setup.py

```diff
@@ -22,8 +22,9 @@
     tests_require=[
         'requests>=2.3.0'
     ],
-
-    author='n.io',
-    author_email='info@n.io',
-    url='http://n.io'
+	
+    licence='Proprietary',
```

Comment:
We should check on this, I think we may want to use a specific license. Unless you've checked with Tim already and this is what we want?

Reply:
Should probably add the classifiers for this as well.

https://pypi.org/pypi?%3Aaction=list_classifiers

Reply:
Confirmed with Tim, this should be `Apache 2.0` license. We can update that in the setup.py and elsewhere in the repo as needed